source/index.txt

```diff
@@ -13,25 +13,25 @@ MongoDB {+driver-short+} Documentation
 .. toctree::
 
    Get Started </get-started>
-   /connect
-   /databases-collections
-   /write-operations
-   /read
-   /run-command
-   /indexes
-   /aggregation
-   /security
-   /data-formats
-   /tools
-   /faq
-   /troubleshooting
-   /whats-new
-   /upgrade
-   /motor-async-migration
-   /pymongo-to-async-guide
-   /previous-versions
-   /issues-and-help
-   /compatibility
+   Connect to MongoDB </connect>
+   Databases & Collections </databases-collections>
+   Write Data </write-operations>
+   Read Data </read>
+   Run a Database Command </run-command>
+   Indexes </indexes>
+   Aggregation </aggregation>
+   Security </security>
+   Data Formats </data-formats>
+   Third-Party Tools </tools>
+   FAQ </faq>
+   Troubleshooting </troubleshooting>
+   What's New </whats-new>
+   Upgrade </upgrade>
+   Migrate Motor to PyMongo Async </motor-async-migration>
```

Comment:
Feel like meaning is lost here without the word 'Migrate' because the 'Motor' library name seems a bit ambiguous to me. However, it does make the title span two lines. Let me know what you think; can be removed if necessary.

Reply:
What about "Migrate from Motor"? The PyMongo part seems implied

Reply:
Same thoughts as Mike. :)